make-uniform/production/compare_against_previous.R

```diff
@@ -47,11 +46,26 @@ find_differences <- function(anti_outcomes_df, diffed_df) {
   
 }
 
-anti_df <- anti_join(previous_df, current_df, by = c("unique_ID"))
-diff_df <- find_differences(anti_df, current_df)
+# do both ways
+anti_previous_df <- anti_join(previous_df, current_df, by = c("unique_ID"))
+diff_previous_df <- find_differences(anti_df, current_df)
 
-relevant_df <- diff_df %>%
-  filter(!(previous_outcome == "missing" & current_outcome == "NA"))
+anti_current_df <- anti_join(current_df, previous_df, by = c("unique_ID"))
+diff_current_df <- find_differences(anti_current_df, previous_df)
 
-table(thin_df$var_name)
+# update the Caltrain IDs and do again
+update_current_df <- current_df %>%
```

Comment:
@jhelsel11: you are correct re: Caltrain bug. When the `ID`s are updated to remove the leading "S", the previous and current data sets match.